README.md

```diff
@@ -89,6 +89,36 @@ export default {
 </script>
 ```
 
+## Using another instance of highlight.js
+
+If you are using a different version of `highlight.js` package than this package, you may end up with two different instances of the `hljs` singleton. You can explicitly inject your instance of `hljs` into this plugin as follows:
+
+```vue
+<script lang="ts">
+import { defineComponent, provide } from 'vue'
+import hljsPlugin from '@highlightjs/vue-plugin'
+import hljs from 'highlight.js'
+
+export default defineComponent({
+    setup() {
+        provide(hljsPlugin.injectionKey, hljs)
```

Comment:
I don't love the naming, is there some Vue convention at work here?  What about `hljsPlugin.hljsInstance` or some such?  I'm not sure the naming makes it clear WHAT you're injecting.

Reply:
There's no particular Vue convention for this. Maybe `HLJS_INSTANCE` to denote what it's providing and in uppercase to denote it's a constant?